apps/desktop/tsconfig.json

```diff
@@ -7,7 +7,9 @@
 		"checkJs": true,
 		"esModuleInterop": true,
 		"forceConsistentCasingInFileNames": true,
+		"noUncheckedIndexedAccess": true,
```

Comment:
yess!!!

Reply:
noUncheckedIndexAccess should be a default option in my mind. I'm for it all the way.

I'm not quite so familiar with verbatimModuleSyntax but it seems to be the new recommended option, so I'm happy to go for it